redis-om-spring/src/main/java/com/redis/om/spring/metamodel/SearchFieldAccessor.java

```diff
@@ -0,0 +1,34 @@
+package com.redis.om.spring.metamodel;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class SearchFieldAccessor {
+  private List<Field> fields = new ArrayList<>();
+  private String searchAlias;
+  private Class<?> targetClass;
+  private Class<?> declaringClass;
+
+  public SearchFieldAccessor(String searchAlias, Field... fields) {
+    this.searchAlias = searchAlias;
+    this.fields.addAll(Arrays.asList(fields));
+    this.targetClass = this.fields.get(0).getType();
+    this.declaringClass = this.fields.get(0).getDeclaringClass();
+  }
+
+  public Field getField() {
+    return fields.get(0);
+  }
+
+  public String getSearchAlias() {
+    return searchAlias;
+  }
+
+  public Class<?> getTargetClass() {
+    return targetClass;
+  }
+
+  public Class<?> getDeclaringClass() { return declaringClass; }
```

Comment:
Consistency is the hobgoblin of a small mind.

Reply:
New IDE doesn't autoformat on save :-( Need to find where to turn that on.

Reply:
Fixed!